diff --git a/Scripts/My First Javascript Test Case/Script1652936119726.groovy b/Scripts/My First Javascript Test Case/Script1652936119726.groovy
index 0625ec8..4381c81 100644
--- a/Scripts/My First Javascript Test Case/Script1652936119726.groovy	
+++ b/Scripts/My First Javascript Test Case/Script1652936119726.groovy	
@@ -17,7 +17,7 @@
 Katalon.onReady(async (driver) => {
   console.log('\r\n--- Execute My First "Hello World" Test! ---\r\n'.yellow);
 
-  await WebUI.comment('Hello! This is my first test! /=)');
+  await WebUI.comment('zHello! This is my first test! /=)');
 
   await WebUI.openBrowser('');
 
